src/bootstrap.rs

```diff
@@ -104,6 +117,8 @@ pub fn initialize() -> windows::Result<()> {
     })
 }
 
+/// Locates the Windows App framework package compatible with the (currently internal)
```

Comment:
Believe this is the more accurate name. (Can you also edit the one above?)

```suggestion
/// Locates the Windows App SDK framework package compatible with the (currently internal)
```

Reply:
Just a reminder. And few lines up needs the same edit.